File: backend/ambevtech/src/main/java/com/ambevtech/core/entity/dto/TempoAtualDTO.java
package com.ambevtech.core.entity.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.io.Serializable;
import java.math.BigDecimal;
import java.util.List;

@Getter
@Setter
@AllArgsConstructor
public class TempoAtualDTO implements Serializable {

    private Long dt;

    private BigDecimal temp;

    private Integer pressure;

    private Integer humidity;

    private List<ClimaDTO> weather;

}

File: frontend/src/app/model/TempoAtualDTO.ts
import { ClimaDTO } from "./ClimaDTO";

export class TempoAtualDTO {

    dt: number;
    temp: number;
    pressure: number;
    humidity: number;
    weather: Array<ClimaDTO>;

    constructor(){
        this.weather = new Array<ClimaDTO>();
    }

}